src/libNode/MicroBlockPreProcessing.cpp

```diff
@@ -287,6 +287,8 @@ void Node::ProcessTransactionWhenShardLeader() {
 
   DetachedFunction(1, txnProcTimer);
 
+  this_thread::sleep_for(chrono::milliseconds(100));
```

Comment:
Maybe make a definition of delay inside constant file?

Reply:
This one is simply for making a small time offset to make the conditional variable declared before the function end, no much meaning to make it configurable.